Core/HW/StereoResampler.cpp

```diff
@@ -248,7 +263,8 @@ unsigned int StereoResampler::Mix(short* samples, unsigned int numSamples, bool
 	// Flush cached variable
 	m_indexR.store(indexR);
 
-	return realSamples / 2;
+	// TODO: What should we actually return here?
```

Comment:
I do think actually produced samples is best, then padding can be decided per platform.

-[Unknown]